genome_designer/utils/import_util.py

```diff
@@ -878,16 +879,22 @@ def import_variant_set_from_vcf(ref_genome, variant_set_name, variant_set_file):
         variant_set_name: Name of the variant set (label).
         variant_set_file: Path to the variant set on disk.
     """
-    # For now, variant set name must be unique even among diff ref genomes.
-    variant_set_name_exists = bool(VariantSet.objects.filter(
-            reference_genome=ref_genome,
-            label=variant_set_name).count())
-    assert not variant_set_name_exists, 'Variant set name must be unique.'
-
-    # Create the VariantSet.
-    variant_set = VariantSet.objects.create(
-            reference_genome=ref_genome,
-            label=variant_set_name)
+    if add_to_existing_set:
```

Comment:
Can you change this to `require_variant_set_does_not_exist` and fix the logic to match this name change?